src/universal_transfer_operator/data_providers/database/google/bigquery.py

```diff
@@ -155,6 +240,209 @@ def truncate_table(self, table):
         """Truncate table"""
         self.run_sql(f"TRUNCATE {self.get_table_qualified_name(table)}")
 
+    def is_native_autodetect_schema_available(self, file: File) -> bool:
+        """
+        Check if native auto-detection of schema is available.
+
+        :param file: File used to check the file type of to decide whether there is a native auto-detection available.
+        """
+        supported_config = self.NATIVE_AUTODETECT_SCHEMA_CONFIG.get(file.location.location_type)
+        if supported_config and file.type.name in supported_config["filetype"]:  # type: ignore
+            return True
+        return False
+
+    def create_table_using_native_schema_autodetection(
+        self,
+        table: Table,
+        file: File,
+    ) -> None:
```

Comment:
Should we change it to the same type as `supported_config["method"](table=table, file=file)`? https://github.com/astronomer/apache-airflow-provider-transfers/pull/66/files#diff-3e26a232c7f20fbe0bf500bde1ce9ef087ef4422e4aecd83b9dc954d112baf1cR267

Reply:
https://github.com/astronomer/apache-airflow-provider-transfers/pull/66/commits/e83030f053afc687621114a710444fdaf51b3a0c